pkg/authorization/api/synthetic.go

```diff
@@ -2,9 +2,10 @@ package api
 
 // Synthetic authorization endpoints
 const (
-	DockerBuildResource = "builds/docker"
-	SourceBuildResource = "builds/source"
-	CustomBuildResource = "builds/custom"
+	DockerBuildResource   = "builds/docker"
+	SourceBuildResource   = "builds/source"
+	CustomBuildResource   = "builds/custom"
+	ExternalBuildResource = "builds/external"
```

Comment:
yes I thought i refreshed before i started reviewing, but in the middle of my review github informed me there was a new commit.